plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/js/WSDLOperations.js

```diff
@@ -0,0 +1,292 @@
+ * initial WSDL DOM from the dummy wsdl(wsdlDom) and the current task
+ * name(currentTaskName) should be provided.
+ * 
+ */
+function generateInputWSDL(wsdlDom, currentTask) {
+    currentTaskName = currentTask + "wrapper";
+    // Get internal parameters
+    serviceURL = $('#' + currentTaskName + ' #taskCallbackServiceURL').val();
+    operationName = $('#' + currentTaskName + ' #taskCallbackOperationName')
+        .val();
+    serviceName = $('#' + currentTaskName + ' #taskCallbackServiceName').val();
+    portName = currentTask + "Port";
+    inputElements = $('#' + currentTaskName + ' #inputmappingTable tr');
+
+    // get Definitions node and Schema node from dummy wsdl dom
+    wsdlSchema = wsdlDom.getElementsByTagName("schema")[0];
+    wsdlDefinitions = wsdlDom.getElementsByTagName("definitions")[0];
+
+    // create Data type elements
+    newComplexType = wsdlDom.createElementNS(
+        "http://www.w3.org/2001/XMLSchema", "xsd:complexType");
+    newComplexType.setAttribute("name", currentTask + "DataType"); // Data type
+    // Name
+    newSequence = wsdlDom.createElementNS("http://www.w3.org/2001/XMLSchema",
+        "xsd:sequence");
+
+    // Loop towards the rows of input data mapping table
+    inputElements.each(function(i, obj) {
+        if (i != 0) { // skip header
+            k = $(this).find("input#taskInputMappingid").val();
+            type = "xsd:" + $("#" + currentTaskName + " #taskInputMappingType" + k)
+                .val();
+            newSubElement = wsdlDom.createElementNS(
+                "http://www.w3.org/2001/XMLSchema", "xsd:element");
+            newSubElement.setAttribute("name", $(this).find(
+                "input#taskInputMappingElementName" + k).val()); // Data
+            newSubElement.setAttribute("type", type); // Data element (Input
+            newSequence.appendChild(newSubElement);
+        }
+    });
+    newComplexType.appendChild(newSequence);
+
+    // create Element element
+    newElement = wsdlDom.createElementNS("http://www.w3.org/2001/XMLSchema",
+        "xsd:element"); // scheme.appendchild()
+    newElement.setAttribute("name", currentTask + "Data");
+    newElement.setAttribute("type", "tns:" + currentTask + "DataType");
+    wsdlSchema.appendChild(newElement);
+    wsdlSchema.appendChild(newComplexType);
+
+    // create Message element
+    newMessage = wsdlDom.createElementNS("http://schemas.xmlsoap.org/wsdl/",
+        "wsdl:message");
+    newMessage.setAttribute("name", currentTask + "Request");
+    newPart = wsdlDom.createElementNS("http://schemas.xmlsoap.org/wsdl/",
+        "wsdl:part");
+    newPart.setAttribute("name", newMessage.getAttribute("name"));
+    newPart.setAttribute("element", "tschema:" + newElement.getAttribute("name")); // Data binding
+    newMessage.appendChild(newPart);
+    wsdlDefinitions.appendChild(newMessage);
+
+    // create portType element
+    newPortType = wsdlDom.createElementNS("http://schemas.xmlsoap.org/wsdl/",
```

Comment:
"http://schemas.xmlsoap.org/wsdl/"
Following string has been used in many places over the script. It's better to create a constant variable and use it where necessary.